webpack/configUtils.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Create a regular expression matcher for filename extensions.
+ */
+const createFnExtMatcher = (...fileExtensions) => {
+  if (typeof fileExtensions !== 'object' || !Array.isArray(fileExtensions)) {
+    throw new Error('`fileExtensions` is not an array');
+  }
+
+  const matcher = new RegExp(`\.(${ fileExtensions.join('|') })$`, 'i');
```

Comment:
`fileExtensions` items should be escaped (e.g., calling the function with arguments `"false|extension", js"` will result in following RegExp: `/.(false|extension|js)$/i`. That would match files with extension false, extension, and js)